version.php

```diff
@@ -23,7 +23,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2016011300;
+$plugin->version   = 2016042500;
```

Comment:
Upgrade went fine on my test site. Both the database upgrade and adding of the new task were successful.